guides/common/modules/con_package-mode-and-image-mode-hosts.adoc

```diff
@@ -0,0 +1,25 @@
+[id="package-mode-and-image-mode-hosts_{context}"]
+= Package mode and image mode hosts
+
+You can operate hosts in two modes: package mode and image mode.
```

Comment:
```suggestion
You can operate hosts in two modes: package mode and image mode.
```
I'd drop this sentence because this can be inferred from the overall context.

Reply:
I'm not sure about this.